org.videolan.VLC.json

```diff
@@ -39,8 +39,33 @@
     "rm -f /app/lib/vlc/plugins/plugins.dat",
     "/app/lib/vlc/vlc-cache-gen /app/lib/vlc/plugins"
   ],
+  "build-options": {
+    "cflags": "-O2 -g -fstack-protector-strong -D_FORTIFY_SOURCE=2",
+    "cxxflags": "-O2 -g -fstack-protector-strong -D_FORTIFY_SOURCE=2",
+    "ldflags": "-fstack-protector-strong -Wl,-z,relro,-z,now -L/app/lib",
+    "env": {
+      "V": "1"
+    }
+  },
   "modules": [
-    "shared-modules/udev/udev-175.json",
+    {
+      "name": "eudev",
```

Comment:
I'd also keep the original `udev` dependency here. I don't think `eudev` has any advantage inside a sandbox compared to `udev` itself.

Reply:
Fine for me, removed.